deploy/00_resolve_keep_token.ts

```diff
@@ -16,8 +16,13 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     // Save deployment artifact of external contract to include it in the package.
     await deployments.save("KeepToken", KeepToken)
   } else if (
-    hre.network.name !== "hardhat" ||
-    (hre.network.config as HardhatNetworkConfig).forking?.enabled
+    hre.network.name === "hardhat" &&
```

Comment:
Same comments as in `deploy/00_resolve_keep_registry.ts`.

Reply:
pls see above